shared/src/map/layers/tiled/vector/symbol/Tiled2dMapVectorSymbolObject.cpp

```diff
@@ -102,7 +103,12 @@ Tiled2dMapVectorSymbolObject::Tiled2dMapVectorSymbolObject(const std::weak_ptr<M
     const bool hasText = !fullText.empty();
     const size_t contentHash = usedKeys.getHash(evalContext);
```

Comment:
This `currentHash` can be relatively expensive and can be moved into the `if (useCustomCrossTileIdentifier ...` branch.